src/pgduckdb_duckdb.cpp

```diff
@@ -167,17 +161,19 @@ DuckDBManager::LoadSecrets(duckdb::ClientContext &context) {
 		DuckDBQueryOrThrow(context, secret_key->data);
 
 		pfree(secret_key->data);
-		secret_id++;
+		++secret_id;
```

Comment:
nit: why swap this around? It has no functional difference here right? I checked to see if this is common duckdb style, and generally it seems postfix is preferred:
```
duckdb on  main via △ v3.28.3
❯ rg '\+\+\w+' | wc -l
3766

duckdb on  main via △ v3.28.3
❯ rg '\w+\+\+' | wc -l
10928
```

Same for postgres:
```
postgres-17 on  REL_17_STABLE [$]
❯ rg '\+\+\w+' | wc -l
1014

postgres-17 on  REL_17_STABLE [$]
❯ rg '\w+\+\+' | wc -l
11167
```

Unless there's a reason that I'm not seeing for doing it this way, I'd prefer to keep it this way. Partially to keep the diff small (to simplify reviews and reduce merge conflicts). And partially because that's what I'm used to.

Reply:
Same for a few lines below